zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -2460,6 +2498,51 @@ components:
           description: The key of the first process instance the message correlated with
           type: integer
           format: int64
+    MessagePublicationRequest:
+      type: object
+      properties:
+        name:
+          description: The name of the message.
+          type: string
+        correlationKey:
+          description: The correlation key of the message.
+          type: string
+          default: ""
```

Comment:
❓ Do we need this? We could also remove the default, remove it from the required list, and form an empty String by default in the backend, right?

Reply:
There is no difference on setting the default here or in the service IMO, using the specification to underline this behaviour make it more clear also for the users

Reply:
Yeah, I don't have a strong opinion here, to be honest. Both is fine for me 👍